automation_tools/__init__.py

```diff
@@ -1173,7 +1173,6 @@ def ak_install(admin_password=None, clean_beaker=True,
 
     # Clean up and install with basic packages.
     clean_rhsm()
-    update_basic_packages()
```

Comment:
update_basic_packages function takes care of it, hence the previous code is alright.

update_basic_packages, installs calls as below,

a) subscribe
b) install basic packages
c) unsubscribe